style-guide.md

```diff
@@ -620,3 +620,45 @@ things = create_things()
 
 # res = triple_len(things)
 ```
+
+# Docstrings
+
+## Preferred delimiters
+
+You **should** enclose docstrings with `"""` delimiters.  That is, you **should
+not** enclose docstrings with `'''` delimiters.
+
+## Layout
+
+You **should** lay out docstring content according to the conventions defined
+in [PEP 257](https://www.python.org/dev/peps/pep-0257/) and [PEP
+287](https://www.python.org/dev/peps/pep-0287/).  Where those two PEPs
+conflict, precedence is given to the recommendations of [PEP
+287](https://www.python.org/dev/peps/pep-0287/).
+
+## Exceptions to docstring style PEPs
+
+The aforementioned [PEP 257](https://www.python.org/dev/peps/pep-0257/)
+recommends that a summary of a method's functionality should appear on the
+first line of a docstring immediately following the opening `"""` delimiter.
+If a team member so wishes, this summary **may** appear on the line immediately
+following the opening `"""` delimiter.  For example,
+
+```python
+# This is fine
+def foo():
+    """Summary of foo's functionality.
+
+    Longer description of foo's functionality.
+    """
+    pass
+
+# This is also fine
+def foo():
+    """
```

Comment:
:+1:  This is the style that is being used in Trinity, Py-EVM and all the rest of the libraries that I've worked with so far.

Reply:
Yep that's the one I prefer. Let's make that the requirement.